Dockerfile

```diff
@@ -16,7 +16,8 @@ FROM gcr.io/heptio-images/golang:1.9-alpine3.6
 MAINTAINER Timothy St. Clair "tstclair@heptio.com"
 
 RUN apk add --no-cache ca-certificates
-ADD sonobuoy /sonobuoy 
+ADD sonobuoy /sonobuoy
```

Comment:
any reason this isn't going it `/bin`?

Reply:
No logical reason.  I'd be in favor of /usr/local/bin/ for the sake of consistency.

Reply:
Sure, I'll make a ticket to improve consistency in our docker builds.

https://github.com/heptio/sonobuoy/issues/254